### PR TITLE
Extract markdown files

### PR DESCRIPTION
**Pull request by theimowski**

### CODE REVIEW

**Comment by forki** [comment]
cool.

could you please check https://github.com/fsprojects/Paket/blob/master/.gitignore#L185 as well?

File: docs/content/commands/update.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Updating a single package
+
+It's also possible to update only a single package and to keep all other dependencies fixed:
```
Comment (forki) [inline docs/content/commands/update.md]: where ist the `paket update` case which updates everything? did we lose it?

**Comment by forki** [comment]
also the contributin guide in the readme needs to be updated.

File: docs/content/commands/update.md (hunk 1 above)
Comment (theimowski) [inline docs/content/commands/update.md]: it's covered by the auto-gen syntax

Comment (theimowski) [inline docs/content/commands/update.md]: auto-gen gets rendered just above the optional additional markdown

Comment (forki) [inline docs/content/commands/update.md]: Awesome

**Comment by theimowski** [comment]
re .gitignore - what I did is for each command generate auto syntax, and then if a corresponding markdown exists in `content\commands` then append the additional markdown, and finally write the result markdown to `content\paket-<command>.md` before FSharp.Formatting comes in. 
So I guess the ignore for `content\paket-*.md` should remain

**Comment by forki** [comment]
ok. what about the readme?

**Comment by theimowski** [comment]
there it is

**Comment by forki** [comment]
:heart:

**Comment by forki** [comment]
this is really awesome. the command line help feels so much better now. Thanks so much.

/cc @agross please give it a try

**Pull request merged**